packages/core/test/integrations/expocontext.test.ts

```diff
@@ -1,6 +1,6 @@
 import type { Client, Event } from '@sentry/core';
 
-import { EXPO_UPDATES_CONTEXT_KEY, expoContextIntegration } from '../../src/js/integrations/expocontext';
+import { OTA_UPDATES_CONTEXT_KEY, expoContextIntegration } from '../../src/js/integrations/expocontext';
```

Comment:
The lint issue should be fixed with a changed import order
```suggestion
import { expoContextIntegration, OTA_UPDATES_CONTEXT_KEY } from '../../src/js/integrations/expocontext';
```